javascript/ql/src/semmle/javascript/dataflow/DataFlow.qll

```diff
@@ -724,6 +750,17 @@ module DataFlow {
 
   /**
    * A data flow node that reads an object property.
+   *
+   * For example, all the following are property reads:
+   * ```js
+   * obj.f               // member access
+   * obj[e]              // computed member access
```

Comment:
These two still say "member". Perhaps "property access expression"?

Reply:
I think I prefer `property access` and `computed property access`.